import-service/handlers/importProductsFile.test.ts

```diff
@@ -0,0 +1,26 @@
+import { APIGatewayEventDefaultAuthorizerContext, APIGatewayProxyEventBase, APIGatewayProxyResult } from 'aws-lambda';
+import { mock, restore } from 'aws-sdk-mock';
+import { importProductsFile } from './importProductsFile';
+
+describe('importProductsFile', () => {
+    const signedUrl: string = 'signedUrl';
```

Comment:
Type declaration is useless in the case.

Reply:
I'm working on project for more than year, where type declaration is mandatory :) it's a habit already